# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/libcubeb/moz.yaml
# Commit: 516cce7af561
# Full Hash: 516cce7af5615e5cb6147cf2f5fa87b4a0d27105
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-02-14 17:00:27
# Regressor Bug: 1527659
# File Overlap Count: 2
# Description:
#   Bug 1527659 - Update cubeb from upstream to 3afc335. r=kinetik
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D19676
# ==============================================================================

diff -r 4858ed100ad4 -r 516cce7af561 media/libcubeb/moz.yaml
--- a/media/libcubeb/moz.yaml	Thu Feb 14 08:27:36 2019 +0000
+++ b/media/libcubeb/moz.yaml	Wed Feb 13 21:50:03 2019 +0000
@@ -19,5 +19,5 @@
   license: "ISC"
 
   # update.sh will update this value
-  release: "feec7e2e893c6dc4188fcb95e15dcf8c19890f46 (2019-01-23 17:15:35 +0200)"
+  release: "3afc3350063ce1d68bbd0287fde4f70b4803d2eb (2019-02-13 10:28:58 +1300)"
 